dev/app-shell/elements/arc-app.html

```diff
@@ -566,6 +566,18 @@
     _onSuggest(e, suggestion) {
       this._setState({plan: suggestion.plan});
     }
+    _onMakeSuggestions(e) {
+      let search = e.detail.search;
+      if (search)
+        search = search.trim().toLowerCase();
```

Comment:
Maybe the caller (event-sender) can do this?

Reply:
done